update_bioinfo_deliveries.py

```diff
@@ -1,413 +0,0 @@
-import socket
```

Comment:
Is  this deletion intentional? If so, maybe quota_log.py should be removed as well?